includes/view-ukuupeople.php

```diff
@@ -0,0 +1,110 @@
+<?php
+		$type = get_the_terms( $people_object->ID , 'wp-type-contacts-subtype');
+        $custom = get_post_custom( $id );
+        $contactdetails_keys = array( 'wpcf-phone' => '' , 'wpcf-mobile'  =>'' );
+        if(isset($custom['wpcf-phone'][0]) && !preg_match('/[^0-9]/', $custom['wpcf-phone'][0]) ) {
```

Comment:
There should be space on both the sides of opening and closing braces of if statement.